### ISSUE TITLE
ColorRect.get/set_frame_color to ColorRect.get/set_color

### ISSUE THREAD

**Issue by AlexHolly** [godotengine/godot#9569]
I suggest renaming the following:

ColorRect.set_frame_color -> ColorRect.set_color
ColorRect.get_frame_color -> ColorRect.get_color

same with

ColorPickerButton.set_pick_color -> ColorPickerButton.set_color
ColorPickerButton.get_pick_color -> ColorPickerButton.get_color

**Comment by Noshyaar** [comment]
Controls already have `get_color` for theming. It has to be renamed first.

**Comment by AlexHolly** [comment]
Hmm this kind of shift the issue to `Control.get_color` -> `Control.get_theme_color`

How about 
ColorRect.set_frame_color -> ColorRect.set_color_frame
ColorRect.get_frame_color -> ColorRect.get_color_frame

and

ColorPickerButton.set_pick_color -> ColorPickerButton.set_color_pick
ColorPickerButton.get_pick_color -> ColorPickerButton.get_color_pick

This is more auto completion friendly.

**Comment by akien-mga** [comment]
Godot's API tries to be intelligible in plain English. `set_color_frame` and `set_color_pick` do not respectively "set the color frame" and "set the color pick" as their name suggests. So no, the current names are much better. The autocompletion already does fuzzy matching and will show them if you type `set_color`.

**Comment by AlexHolly** [comment]
In my case the Picker was a global variable so I didn't get any auto completion trying `set_color` as the variable is called color in the inspector results in an error since its called `set_pick_color`.

Is there a way to check for the type and show auto completion on global-script variables?

**Comment by AlexHolly** [comment]
Since there is no good renaming solution I looked a bit into my global script and tried to get some auto completion.

Can anyone explain why this is not working?

I have a global script named "global"

```
var save = preload("res://scenes/Save.tscn").instance()

func _ready():
	#here the auto completion for save works
	save.
```

When I use global.save. on another script it is not showing any auto completion
